### PR TITLE
correct ifa effect

### PR DESCRIPTION
**Pull request by KjellSwedin**

### CODE REVIEW

File: src/vivarium_gates_bep/components/treatment.py [hunk 1]
```diff
@@ -123,9 +126,9 @@ def setup(self, builder):
 
     def adjust_lbwsg(self, index, exposure):
         pop = self.population_view.get(index)
-        baseline_covered = pop[project_globals.BASELINE_COLUMN] == project_globals.TREATMENTS.IFA
-        exposure.loc[baseline_covered, project_globals.BIRTH_WEIGHT] -= self.treatment_effects[project_globals.TREATMENTS.IFA]
 
+        p_ifa = load_ifa_proportion_among_anc(self.draw, self.location)
```
Comment (collijk) [inline src/vivarium_gates_bep/components/treatment.py:130]: Load this once during setup.  No need to repeatedly recompute.

**Comment by collijk** [comment]
Include validation that the effect size is as expected.

File: src/vivarium_gates_bep/components/treatment.py (hunk 1 above)
Comment (KjellSwedin) [reply src/vivarium_gates_bep/components/treatment.py:130]: Done.

**Comment by KjellSwedin** [comment]
exposure.birth_weight.describe()
Out[9]: 
count    10000.000000
mean      2890.227219
std        692.042016
min        100.000000
25%       2563.019160
50%       2916.152312
75%       3325.439873
max       4522.967058
Name: birth_weight, dtype: float64
ifa_covered = pop[project_globals.SCENARIO_COLUMN] == project_globals.TREATMENTS.IFA

// before applying treatment effect
exposure.birth_weight[ifa_covered].mean()
Out[11]: 2873.68788777548
// after applying effect
exposure.birth_weight[ifa_covered].mean()
Out[12]: 2922.1948929206965
2922.1948929206965 - 2873.68788777548
Out[13]: 48.50700514521668    // effect size should be +57.73 g (7.66 to 107.79)

**Pull request merged**